snyk/cli
# chore: Introduce analytics in Extensible CLI

#### What does this PR do?
This PR adds a basic implementation of the analytics module in golang. Therefore introduces a global configuration package to access configuration from different sources in a unified way. 
And some minor refactorings to reduce coupling and avoid cyclic dependencies.

#### Where should the reviewer start?
For the integration of the feature the best location is `cliv2/cmd/cliv2/main.go`.

#### How should this be manually tested?
Run any command and use for example logz.io to see the events arrive. Commands that execute in the legacy CLI will create two analytic entries.

#### What are the relevant tickets?
HMMR-528
